lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -8,6 +8,45 @@
     position: relative;
   }
 
+  &.-fullscreen {
+    position: fixed;
+    z-index: 1000;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+
+    &:before {
+      content: "";
+      display: block;
+      background: url('../logo.png');
+      background-size: 72px 60px;
+      width: 72px;
+      height: 60px;
+      position: absolute;
+      top: 0;
+      left: 0;
+      z-index: 1;
+      margin: 12px;
+    }
+
+    > .wrapper {
+      height: inherit;
+      max-width: none;
+      padding-top: 64px;
+    }
+
+    .slideshow {
+      position: relative;
+      top: 50%;
+      transform: translateY(-50%);
+
+      .slide {
```

Comment:
we use `slideshow__slide` for the specific slide and it already should have a `display: block` declaration...